packages/modeling/src/maths/OrthoNormalBasis.js

```diff
@@ -106,7 +106,7 @@ OrthoNormalBasis.GetCartesian = function (xaxisid, yaxisid) {
   } else {
     throw new Error('OrthoNormalBasis.GetCartesian: invalid combination of axis identifiers. Should pass two string arguments from [X,Y,Z,-X,-Y,-Z], being two different axes.')
   }
-  return new OrthoNormalBasis(new Plane(new Vector3D(planenormal), 0), new Vector3D(rightvector))
+  // return new OrthoNormalBasis(new Plane(new Vector3D(planenormal), 0), new Vector3D(rightvector))
```

Comment:
Why is the return statement commented out? Isn't this return needed?

Reply:
Actually, this function is dead code, using unknown types, and untested. I'd prefer just to remove the unused code. Thoughts?

Reply:
I removed the unused untested code. This is GIT so the code can be recovered anytime.